king_phisher/client/application.py

```diff
@@ -102,21 +102,21 @@ class KingPhisherClientApplication(_Gtk_Application):
 	__gsignals__ = {
 		'campaign-changed': (GObject.SIGNAL_RUN_FIRST, None, (str,)),
 		'campaign-created': (GObject.SIGNAL_RUN_FIRST, None, (str,)),
-		'campaign-deleted': (GObject.SIGNAL_RUN_FIRST, None, (str,)),
+		'campaign-deleted': (GObject.SIGNAL_RUN_LAST, None, (str,)),
```

Comment:
Since these are run last now, and you can use `connect_after` all of these "deleted" past-tense names should be "delete".